roles/engineblock/tasks/main.yml

```diff
@@ -64,6 +64,30 @@
   when:
     customlogo.stat.exists
 
+- name: Check if we have a custom background back image for the feedback page
+  local_action: stat path="{{ inventory_dir }}/files/eb/background-back.svg"
+  register: eb_customfeedbackbackground
```

Comment:
`customfeedbackbackground` seems odd. I'm not fully familiar with the Ansible naming conventions, but I suspect you are allowed to use underscores to make the register more readable.

`eb_custom_feedback_background`

Reply:
It was shamefully pasted from `customlogo` so I thought that was the used convention.